lagesonum/bottle_app.py

```diff
@@ -30,47 +29,61 @@
 # ('ar_AR', 'Arab'),
 DEFAULT_LOCALE = 'en_US'
 
+
 @route('/')
 @view('start_page')
 def index():
     """1.Seite: Helfer steht am LaGeSo und gibt Nummern ein [_____] """
     return {'entered': []}
 
+
+def get_fingerprint(request):
+    usr_agent = str(request.environ.get('HTTP_USER_AGENT', ''))
+    usr_lang = str(request.environ.get('HTTP_ACCEPT_LANGUAGE', ''))
+    usr_ip = str(request.remote_addr)
+
+    usr_fingerprint = u'{}{}{}'.format(usr_agent, usr_lang, usr_ip)
+
+    return hashlib.md5(usr_fingerprint.encode("utf-8")).hexdigest()
+
+
+def parse_numbers(input_string, first_only=False):
+    r = re.compile(r'[a-z]\s*[0-9]+', re.IGNORECASE)
```

Comment:
what is this `\s*` about? is `A    234` a valid number too? Won't it get split into two parts via the `num.split()` below?